spinn_front_end_common/utilities/helpful_functions.py

```diff
@@ -303,6 +304,19 @@ def convert_vertices_to_core_subset(vertices, placements):
     return core_subsets
 
 
+def find_executable_start_type(machine_vertex, graph_mapper=None):
```

Comment:
I can see how this function would be useful, but is it actually called from anywhere at the moment?

Reply:
ermmmm...... having checked from the ear bitfield's cleaned branches. it seems that's a hang over from the all algorithms. its used in "find application chips used" and "bit_field_compressor_report" so i think its safe to remove it from this version. But its worth noting for its going to cause a merge confict when these divergant bits say hi to each other

Reply:
solved this one quickly